### PR TITLE
Seeing if we can speed up spotbugs

### LINKED ISSUE #4974: Evaluate options to make spotbugs run faster - start with "effort"
Spotbugs may soon become the longest pole in building Track 2 client libraries. This task is to look into "effort" for spotbugs and tune it to improve build times.  https://spotbugs.readthedocs.io/en/stable/effort.html

### PR DESCRIPTION
**Pull request by JonathanGiles**
by lowering the effort and not forking the JVM for each module.

Fixes #4974

### CODE REVIEW

**Comment by JonathanGiles** [comment]
This shaves about 25 seconds off of the Analyze job, so I think it is worth merging.

File: pom.client.xml [hunk 1]
```diff
@@ -164,12 +164,13 @@
           </dependency>
         </dependencies>
         <configuration>
-          <effort>Max</effort>
+          <effort>min</effort>
```
Comment (srnagar) [inline pom.client.xml]: Maybe we should run nightly builds with `max` or `more` and publish the [results here](https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-java/staging/spotbugsXml.html#{%22tab%22:%22summary%22,%22includeFixed%22:false,%22release%22:-1,%22priority%22:4})

Min does very little and may not catch some issues.
https://spotbugs.readthedocs.io/en/stable/effort.html

Comment (JonathanGiles) [reply pom.client.xml]: My expectation is even at min we should get everything we want.

Comment (JonathanGiles) [reply pom.client.xml]: But I should add, there was almost no difference between min and max on my machine. The only win was from not forking the JVM for each module, so I am also happy to return this to max.

**Pull request merged**